helm/hpcc/templates/localroxie.yaml

```diff
@@ -12,14 +12,16 @@ data:
     roxie:
 {{ toYaml (omit .me "logging") | indent 6 }}
 {{- include "hpcc.generateLoggingConfig" . | indent 6 }}
-{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" (list "storage" "ecl" "ecl-user" ) ) | indent 6 }}
+{{ include "hpcc.generateVaultConfig" (dict "root" .root "categories" .secretsCategories ) | indent 6 }}
```

Comment:
I think this is mean to be secretsCategoriesUser?

Reply:
I've remove the need to secretsCategoriesUser - I'm using secretsCategories for everything now.